src/__tests__/old-act.js

```diff
@@ -32,18 +32,15 @@ test('async act works even when the act is an old one', async () => {
     console.error('sigil')
   })
   expect(console.error.mock.calls).toMatchInlineSnapshot(`
-        Array [
-          Array [
-            "sigil",
```

Comment:
It looks like this log is no longer happening. Could you fix your changes so you don't have to change this snapshot?

Reply:
Changed implementation to revert back to this snapshot.